hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/token/TokenPauseSpecs.java

```diff
@@ -98,6 +98,16 @@ final Stream<DynamicTest> cannotPauseWithInvalidPauseKey() {
                 .then(tokenPause(PRIMARY).signedBy(DEFAULT_PAYER, OTHER_KEY).hasKnownStatus(INVALID_SIGNATURE));
     }
 
+    @HapiTest
+    final Stream<DynamicTest> cannotUnpauseWithInvalidPauseKey() {
+        return defaultHapiSpec("cannotUnpauseWithInvalidPauseKey")
+                .given(newKeyNamed(PAUSE_KEY), newKeyNamed(OTHER_KEY))
+                .when(
+                        tokenCreate(PRIMARY).pauseKey(PAUSE_KEY),
+                        tokenPause(PRIMARY).signedBy(DEFAULT_PAYER, PAUSE_KEY))
+                .then(tokenUnpause(PRIMARY).signedBy(DEFAULT_PAYER, OTHER_KEY).hasKnownStatus(INVALID_SIGNATURE));
+    }
```

Comment:
New format preferred over old

```suggestion
    @HapiTest
    final Stream<DynamicTest> cannotUnpauseWithInvalidPauseKey() {
        return hapiSpec(
                newKeyNamed(PAUSE_KEY), newKeyNamed(OTHER_KEY),
                tokenCreate(PRIMARY).pauseKey(PAUSE_KEY),
                tokenPause(PRIMARY).signedBy(DEFAULT_PAYER, PAUSE_KEY),
                tokenUnpause(PRIMARY).signedBy(DEFAULT_PAYER, OTHER_KEY).hasKnownStatus(INVALID_SIGNATURE);
    }
```